### ISSUE TITLE
Cant create reserve proof with message

### ISSUE THREAD

**Issue by MexicanTakeout** [monero-project/monero#8443]
This is on  Monero 'Fluorine Fermi' (v0.18.0.0-b6a029f22) stage net. Steps to reproduce:
- create wallet
- use faucet to supply moneroj to wallet
- open wallet and type in get_reserve_proof 10 "this is a message"
response shows Error: usage: get_reserve_proof (all|<amount>) [<message>]

Strangely enough, it works with single word message but the signature turns out to be bad when check with check_reserve_proof. Else get_reserve_proof  works fine with just the amount.

**Comment by reemuru** [comment]
This is not true. You can in fact create reserve proof with a message. From a technical standpoint there is no reason to have whitespace in a message. Hence change it to `this_is_a_message` with no quotations. You are thinking in terms of 'message' as in terms of a written sentence with proper punctuation.

**Comment by MexicanTakeout** [comment]
RPC call can parse message inside double quotes like here https://i.imgur.com/epIinY6.png
So I think this is an inconsistency issue.

The use of message string also wasn't clear to me at first. I expected it to be like a secret message that would reveal to a receiver after verified proof, but it is more like a password to use in decrypting the reserve proof.
